measures/src/main/java/io/apisense/network/ping/Rtt.java

```diff
@@ -0,0 +1,32 @@
+package io.apisense.network.ping;
+
+/**
+ * Definition of a ping RTT.
+ */
+public class Rtt {
+
+    private final float min;
```

Comment:
Immutability is better, the RTT is the result of the ping so there is no need to modify it, don't you?